docs/i18n/en/docusaurus-plugin-content-docs/current.json

```diff
@@ -50,5 +50,73 @@
   "sidebar.tutorialSidebar.category.cstor": {
     "message": "cstor",
     "description": "The label for category cstor in sidebar tutorialSidebar"
+  },
+  "version.label": {
+    "message": "Version Main",
+    "description": "The label for version current"
+  },
+  "sidebar.sidebar.category.Introduction": {
+    "message": "Introduction",
+    "description": "The label for category Introduction in sidebar sidebar"
```

Comment:
@IsAmrish why the **sidebar** word is repeated twice in every description ??

Reply:
this is auto-generated @AVRahul

Reply:
We are trying to support internationalization (support multiple languages )  but currently only supporting the `english`

Reply:
ok